pennylane/operation.py

```diff
@@ -713,6 +713,17 @@ def inverse(self):
         """Boolean determining if the inverse of the operation was requested."""
         return self._inverse
 
+    def adjoint(self, do_queue=False):
+        """Create an operation that is the adjoint of this one.
```

Comment:
Maybe it would be good to define adjoint here?

Reply:
+1 on defining the adjoint here :P

Reply:
Added. Would we want a link back to wikipedia or something?

Reply:
Maybe just the following?
```suggestion
        """Create an operation that is the adjoint or inverse of this one.
```

That way it will show up if people search the docs for 'inverse'.